src/main/scala/org/biodatageeks/sequila/datasources/VCF/VCFRelation.scala

```diff
@@ -14,16 +17,21 @@ class VCFRelation(path: String)(@transient val sqlContext: SQLContext) extends B
 
   val spark: SparkSession = sqlContext.sparkSession
 
+  val cleanContigUDF = udf[String,String](DataQualityFuncs.cleanContig)
 
-  lazy val df: DataFrame =  spark.read
-    .format("com.lifeomic.variants")
-    .option("use.format.type", "false")
+  lazy val inputDf: DataFrame =  spark
+    .read
+    .format("vcf")
+    .option("splitToBiallelic", "true")
```

Comment:
tu jest dok jak robic normalizacje.
https://glow.readthedocs.io/en/latest/tertiary/variant-normalization.html
jedyna rzecz to to ze tu potrzba indeksu, wiec moglibysmy robic tak ze ustawiac jakis param z lokalizacja indeksu hg.